primer/logo.include

```diff
@@ -0,0 +1,3 @@
+<a href="https://solidproject.org/TR/" class="logo">
+    <img alt="Solid" src="https://solid.github.io/solid-oidc/solid.svg" width="72">
```

Comment:
```suggestion
    <img alt="Solid" src="https://solidproject.org/TR/solid.svg" width="72">
```

@acoburn what do you think about reusing this URL across different document?

Reply:
Yes, I've been trying to switch to that URL